.github/workflows/ci.yml

```
@@ -23,12 +23,14 @@ jobs:
           name: code-coverage
           path: coverage/
 
-  sonar-scan:
+  sonarqube:
+    name: run sonarqube
     if: ${{ github.actor != 'dependabot[bot]' }}
-    needs: [build]
+    needs: [test]
```

```suggestion
    needs: [build]
```
See https://github.com/inrupt/pod-browser/actions/runs/4256962501: the name of the job running the tests in this workflow is `build` (on line 9)